scripts/deploy-test/deploy-gcp.sh

```diff
@@ -0,0 +1,45 @@
+project_id=$(grep '"project_id"' "$(dirname "$0")/deploy-config.json" | sed 's/.*: "\(.*\)".*/\1/')
+access_token=$(grep '"access_token"' "$(dirname "$0")/deploy-config.json" | sed 's/.*: "\(.*\)".*/\1/')
+provider_email=$(grep '"provider_email"' "$(dirname "$0")/deploy-config.json" | sed 's/.*: "\(.*\)".*/\1/')
+environment=$(grep '"environment"' "$(dirname "$0")/deploy-config.json" | sed 's/.*: "\(.*\)".*/\1/')
+ENV=$environment
+
+if [ -f "apps/intake/env/.env.$environment" ]; then
+    first_setup=false
+else
+    first_setup=true
+fi
+
+echo $first_setup
+
+
+if $first_setup; then
+    sh scripts/ottehr-setup.sh $project_id $access_token $provider_email $environment
+else
+    npm install
+fi
+
+cd ./scripts/deploy-test
+ts-node ./gcp/deploy-test.ts
+cd ../..
+npx tsx ./scripts/ottehr-setup.ts
```

Comment:
looks like we conditionally do this above, why definitely do it again?

Reply:
oh these are different scripts. can you add a comment explaining the difference in intent for this line and line 17?

Reply:
this was for testing only